marking-criteria.qmd

```diff
@@ -1,19 +1,20 @@
-
 ---
 title: "Marking Criteria"
-format: html
+format:
+  html:
+    theme: cerulean
 ---
 
 ## Marks
 
 Marks are awarded in 4 categories, accounting for the following criteria:
 
-    **Data processing: 20%**
+### Data processing: 20%
```

Comment:
Much better, was going to mention this, thank you!

Reply:
Thanks for the quick review and approve :)